gee_data_catalogs/dialogs/catalog_dock.py

```
@@ -784,7 +784,9 @@ def _export_image(self):
         except ImportError as e:
             raise ImportError(
                 f"Required packages not available: {e}\n"
-                "Please install xarray and xee: pip install xarray xee"
+                "Open the plugin's Dependencies panel and click Install/Update "
+                "to add xee and xarray, or install them manually with: "
+                "pip install xarray xee"
             )
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)